### PR TITLE
Cu 1e0u13c add time commitment field to job postings gordon wang

### PR DESCRIPTION
**Pull request by G-Wang12**
Made it so that the timecommitment property of the jobpostings will be read and displayed under the 'requirements' section of job postings.

### CODE REVIEW

File: src/components/Postings/TimeCommitment.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (LavanSumanan) [inline src/components/Postings/TimeCommitment.tsx:1]: Remove this unnecessary component

**Review by LavanSumanan — COMMENTED** [review]
Looks good! Just make sure to remove code that isn't being used (e.g. the "TimeCommitment" component

**Comment by G-Wang12** [comment]
Oh yeah I just realized. I committed that part like 3 weeks ago when I
initially started. Thank you for the feedback Lavan!

**Review by LavanSumanan — APPROVED** [review]
lgtm!

**Review by jmawloo — APPROVED** [review]

**Pull request merged**